static/src/javascripts/projects/commercial/modules/dfp/fill-advert-slots.js

```diff
@@ -45,7 +45,7 @@ const fillAdvertSlots = (
     start: () => void,
     stop: () => void
 ): Promise<void> => {
-    if (commercialFeatures.dfpAdvertising) {
+    if (commercialFeatures.dfpAdvertising || commercialFeatures.adFree) {
```

Comment:
I'm not sure if this is needed. Generally, `dfpAdvertising` is on, but is there any situation where `dfpAdvertising` is false, `adFree` is true, and you'd want to fill advert slots?